### ISSUE TITLE
Multi-platform Games

### ISSUE THREAD

**Issue by d4rkd0s** [Speed-Router/speedrouter.run#10]
![image](https://user-images.githubusercontent.com/2348814/34925020-fa65467c-f963-11e7-83c1-e6ff7f5b1a2f.png)

The original plan for speedrouter.run was to separate games per platform but now, after careful consideration, it seems that combining the games and setting the platforms in the "runs" rules, is a far better option.

**Comment by awurster1** [comment]
Could it not be argued that the strats and routes could be different per platform thus drastically changing the times and making it unfair to compare each platform? @d4rkd0s

**Comment by d4rkd0s** [comment]
@awurster1 yes! They are "sometimes" so we kind of have to be able to deal with both cases.
